.travis.yml

```diff
@@ -7,25 +7,43 @@ php:
 env:
   - VERSION=1.7.2.2
     CHANNEL=major
+    TEST_TYPE=standard
```

Comment:
I think "use_channel" or "from_channel" would be more declarative. If you choose "use_" then you should rename "from_archive" to "use_archive"

Reply:
Fixed in https://github.com/PrestaShop/autoupgrade/pull/359/commits/a08ba3e030046b9abaf4c02ff5ca92106d9f175d